### DISCUSSION TITLE
VSCode Webview extension can not work online

### DISCUSSION THREAD

**Discussion by huzhixiao** [eclipse-theia/theia#13538] [Q&A]
https://ws.theia-cloud.io/c32c8318-37f5-4c5e-afc7-538fc77d6809/#/home/project
install GitLens Extension
![image](https://github.com/eclipse-theia/theia/assets/17584383/3567a3ed-02a9-4dd0-8557-4474da5fea5d)
![image](https://github.com/eclipse-theia/theia/assets/17584383/b9502029-16f2-496c-9757-767a20190741)

**Comment by msujew** [comment]
Hey @huzhixiao,

Theia/vscode webviews use a subdomain pattern to generate their URL. If these subdomains aren't forwarded correctly, the webview cannot be displayed.

If you're using Theia cloud, this issue if on them and not on Theia itself. Please create a discussion on the [appropriate repo](https://github.com/eclipsesource/theia-cloud).

Alternatively, see https://github.com/eclipse-theia/theia/discussions/9912#discussioncomment-8586022.
